### DISCUSSION TITLE
pip failed

### DISCUSSION THREAD

**Discussion by lengyuner** [microsoft/onnxruntime#6111] [Other Q&A]
```
pip install -i https://pypi.python.org/simple onnxruntime-gpu
Looking in indexes: https://pypi.python.org/simple
ERROR: Could not find a version that satisfies the requirement onnxruntime-gpu
ERROR: No matching distribution found for onnxruntime-gpu
```

**Comment by snnn** [comment]
Please let us know your python version,pip version and operating system type.

**Comment by XQyangy** [comment]
onnxruntime-gpu don't support macOS

**Reply by dfauvarq** [reply]
I have a similar issue on MacOS with Python3.7

pip3.7 install onnxruntime==1.9.0
ERROR: Could not find a version that satisfies the requirement onnxruntime==1.9.0 (from versions: 0.1.4, 0.2.1, 0.3.0, 0.4.0, 0.5.0, 1.1.0, 1.1.1, 1.3.0, 1.4.0, 1.5.1, 1.5.2, 1.6.0, 1.7.0, 1.8.0, 1.8.1)
ERROR: No matching distribution found for onnxruntime==1.9.0

Do you have plans to make a pip distribution available for macos

**Reply by snnn** [reply]
What if you download "https://files.pythonhosted.org/packages/6a/9e/67cf59f02a8f309413a23699b37e015fdd3871b85472ee5718a9598caa6f/onnxruntime-1.9.0-cp37-cp37m-macosx_10_14_x86_64.whl" and install it directly?

**Reply by itGaodazhuang** [reply]
it showed "ERROR: onnxruntime-1.11.0-cp39-cp39-win_amd64.whl is not a supported wheel on this platform."

**Reply by ptn77** [reply]
anyone able to run this on mac? I'm getting the same error

**Reply by ceekr** [reply]
Likewise. Was installing WhisperX and it aborts with this error. I am on MacOS 13 with Apple M1 Max chips. So, I get that onnxruntime-gpu won't support macOS. But, I was hoping for a workaround. Thanks! Also, as suggested by @snnn, downloaded the [MacOS ARM64 version](https://files.pythonhosted.org/packages/6b/4a/fda588a7a296fa0a5b1618d7de7a1be583125b7f6616463618a092c613de/onnxruntime-1.16.1-cp311-cp311-macosx_11_0_arm64.whl), and tried installing using: 
`pip3 install ~/Downloads/onnxruntime-1.16.1-cp311-cp311-macosx_11_0_arm64.whl `

Resulting in an error: ERROR: `onnxruntime-1.16.1-cp311-cp311-macosx_11_0_arm64.whl is not a supported wheel on this platform.`

So does not look like that will work on Apple M1 Max chipset.
